src/changelog.ts

```diff
@@ -43,7 +53,8 @@ export class Changelog<M extends object> {
       await this.observer(entity, schema)
 
       this.transactionManager.commit(id, new Entity(entity).envelope())
-
+      this.cache.set(id, entity)
```

Comment:
Do we need the entire entity in the cache?

Reply:
no, I changed to save only the entity id